src/coreComponents/linearAlgebra/interfaces/hypre/HypreMatrix.cpp

```diff
@@ -787,7 +787,7 @@ void HypreMatrix::scale( real64 const scalingFactor )
   HYPRE_Int const diag_nnz = hypre_CSRMatrixNumNonzeros( prt_diag_CSR );
   HYPRE_Real * const ptr_diag_data = hypre_CSRMatrixData( prt_diag_CSR );
 
-  forAll< execPolicy >( diag_nnz, [=] GEOSX_HOST_DEVICE ( HYPRE_Int const i )
+  forAll< hypre::execPolicy >( diag_nnz, [=] GEOSX_HYPRE_HOST_DEVICE ( HYPRE_Int const i )
```

Comment:
where is `hypre::execPolicy` defined and how is it set?

Reply:
Here: https://github.com/GEOSX/GEOSX/blob/8cd1d4c98912630f0b418c3086cfab48106032c3/src/coreComponents/linearAlgebra/interfaces/hypre/HypreUtils.hpp#L67-L73

Reply:
oh...duh.